src/status_im/ui/screens/popover/views.cljs

```diff
@@ -51,28 +52,36 @@
                                      (show-panel-anim bottom-anim-value alpha-value))
 
                                  :else
-                                 (do (reset! clear-timeout (js/setTimeout #(reset! current-popover nil) 500))
```

Comment:
why removing this?

Reply:
I moved the item in the `do block` to :on-press
even
```clojure
(reset! current-popover nil)
```
in its place is redundant as it will already be `nil`

Reply:
oh.. and because of this comment by @errorists 
> The QR popover has this quirk to it, on dismiss it shrinks down (or rather it looks like the QR is removed once you tap dismiss and the popover recalculates its size based on that). We only want to animate the translateY position.